CHANGELOG.md

```diff
@@ -67,6 +67,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - `Add` and `Subtract` keys are now named `NumpadAdd` and `NumpadSubtract` respectively
 - Feature checking when cross compiling between different operating systems
 - Crash when writing to the clipboard fails on Wayland
+- `SpawnNewInstance` will now remain in the same working directory on macOS
```

Comment:
I don't really like the way it's written here.

Maybe something like
```
`SpawnNewInstance` not using working directory of active process on macOS
```

Reply:
The way you rephrased it is a bugfix, which it isn't. This should actually be in the changed or added section.

Besides that, "will now remain" is a bit misleading, since it doesn't really remain anywhere. It changes into that directory.

Reply:
I put the change log entry in the "Fixed" section because it's clearly stated behaviors wasn't being respected in from the configuration:

https://github.com/alacritty/alacritty/blob/80688e082d13197b6661cfb6cfa60734034270cf/alacritty.yml#L375-L379

That said, I'll move this to the changed section, since that's applicable as well, and the remaining open issue (#1979) is marked as an enhancement.

Reply:
Thanks for the clarification, but at that point I'd say it's more of a bug in our documentation than anything else. This definitely was planned to work exactly the way it does right now on macOS. Generally unless it's actually something that should be behaving differently, I wouldn't consider it a bug.

Having things fall in the `Fixed` section is also usually just the "goto" place for everything that isn't a new feature. So if there's any reason to move it somewhere else it's probably a good idea to do so.